docs/using/api.ipynb

```diff
@@ -762,14 +762,22 @@
     "  (only selected metadata can be merged if `cell_meta` is not `None`)\n",
     "  \n",
     "In this way we create a notebook that is *fully* up-to-date for both its code and textual content:"
-   ]
+   ],
+   "metadata": {}
   },
   {
    "cell_type": "code",
    "execution_count": 26,
-   "metadata": {},
+   "source": [
+    "cache.merge_match_into_file(\n",
```

Comment:
does this update the cache itself, or simply return a notebook that is merged? We should make this clear via a `note` or something